qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,17 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
+                index = self.currentIndex()
+                if index != -1:
```

Comment:
If you do:

```suggestion
                if index == -1:
                    return
```

here instead, you can dedent the rest by one, making this a bit simpler to read.